modules/security/pki/default.nix

```
@@ -10,20 +10,27 @@ let
   };
 
   caCertificates = pkgs.runCommand "ca-certificates.crt"
-    { files =
+    {
+      files =
         cfg.certificateFiles ++
         [ (builtins.toFile "extra.crt" (concatStringsSep "\n" cfg.certificates)) ];
-     }
+    }
     ''
       cat $files > $out
     '';
 in
 
 {
   options = {
+    security.pki.enable = mkOption {
```

Can you change the name to match the NixOS option?

```suggestion
    security.pki.installCACerts = mkOption {
```

---

Sure!